### PR TITLE
Improve test coverage

### PR DESCRIPTION
**Pull request by lwrubel**
## Why was this change made? 🤔
Adding another test, since what I did before was not really testing certain errors. 

## How was this change tested? 🤨
Unit

⚡ ⚠ If this change has cross service impact, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test) that use sdr-api*** (e.g.create_object_h2_spec.rb) and/or test in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: lib/globus/client/endpoint.rb [hunk 1]
```diff
@@ -80,6 +80,7 @@ def call_mkdir(path)
         response
       end
 
+      # Makes the API call to Globus to set permissions
```
Comment (justinlittman) [inline lib/globus/client/endpoint.rb:83]: Typo.

**Review by justinlittman — APPROVED** [review]
Approve pending typo fix.

File: lib/globus/client/endpoint.rb (hunk 1 above)
Comment (lwrubel) [reply lib/globus/client/endpoint.rb:83]: What is the typo?

**Review by mjgiarlo — APPROVED** [review]

File: lib/globus/client/endpoint.rb (hunk 1 above)
Comment (mjgiarlo) [reply lib/globus/client/endpoint.rb:83]: I don't see a typo either. I'll leave to @justinlittman to merge if he concurs

**Comment by lwrubel** [comment]
Rebased.

**Comment by mjgiarlo** [comment]
@lwrubel If you have style errors now, `standardrb --fix` should fix 'em. (or its rubocop equivalent.)

**Comment by mjgiarlo** [comment]
I'm going to merge and commit to fixing the typo if there still is one, just to keep things moving.

**Pull request merged**